sidecar/config/config.go

```diff
@@ -146,6 +159,9 @@ type Config struct {
 	Supervise bool `yaml:"supervise"`
 
 	HealthChecks []HealthCheck `yaml:"healthchecks"`
+	HealthChecksCACertPath string `yaml:"healthchecks_ca_cert_path"`
```

Comment:
I was really trying to cut down on the number of environment variables needed. We are trying to force people to using the YAML config for most part instead of creating umpteen number of CLI flags, as it tends to get more confusing.

In this case, I would say, yes. We are probably forcing people to use YAML config if they want to use HTTPs health checks reliably.

@zcahana @gicaceres @ijsnellf any thoughts?